web/sites/dev

```diff
@@ -0,0 +1 @@
+default/
```

Comment:
Good question, it's a symlink to the `default` sites folder. Since we have `live` and `test` folders, I thought we should have `dev`, too.